### PR TITLE
[BUGFIX] Fixed issue where CLI checkpoint list truncated names (#1518)

### PR DESCRIPTION
**Pull request by eugmandel**
Changes proposed in this pull request:
- Fix issue #1518 - "rstrip" was used to remove the "yml" file extension.

Previous Design Review notes:
- None

Thank you for submitting!

### CODE REVIEW

**Review by jcampbell — APPROVED** [review]
Suggestion inline, available to just merge.

File: docs/reference/changelog.rst [hunk 1]
```diff
@@ -9,6 +9,7 @@ develop
 
 * [BUGFIX] Fixed bug preventing GCS Data Docs sites to cleaned
 * [BUGFIX] Correct doc link in checkpoint yml
+* [BUGFIX] Fixed #1518
```
Comment (jcampbell) [inline docs/reference/changelog.rst]:
```suggestion
* [BUGFIX] Fixed issue where CLI checkpoint list truncated names (#1518)
```

**Comment by jcampbell** [comment]
Also, looks like linter needs to run.

**Review by Aylr — APPROVED** [review]
🤦 Curse you `.rstrip()`!

**Comment by jcampbell** [comment]
ping for linting...

**Pull request merged**